pages/instructorAssessmentGroups/instructorAssessmentGroups.js

```diff
@@ -0,0 +1,205 @@
+const ERR = require('async-stacktrace');
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
+
+const error = require('@prairielearn/prairielib/error');
+const groupUpdate = require('../../lib/group-update');
+const sqldb = require('@prairielearn/prairielib/sql-db');
+const sqlLoader = require('@prairielearn/prairielib/sql-loader');
+
+const sql = sqlLoader.loadSqlEquiv(__filename);
+function obtainInfo(req, res, next){
```

Comment:
Is it possible to write a more descriptive name for this function?

Reply:
^ ditto or write a jsdoc comment